swift/ql/lib/codeql/swift/dataflow/internal/FlowSummaryImplSpecific.qll

```diff
@@ -110,6 +110,11 @@ private string getContentSpecific(ContentSet cs) {
     result = "Field[" + c.getField().getName() + "]"
   )
   or
+  exists(Content::EnumContent c |
+    cs.isSingleton(c) and
+    result = "EnumElement[" +  c.getSignature() + "]"
+  )
+  or
```

Comment:
Thanks for adding this, I missed it in https://github.com/github/codeql/pull/13795 .